src/kostentraeger/transformer.ts

```diff
@@ -102,6 +125,10 @@ function createReceiptTransmissionMethods(uemList: UEM[], dfuList: DFU[]): Recei
         ftam = dfuList.find((dfu) => dfu.dfuProtokollSchluessel == "016")?.address
     }
 
+    if(!paperReceipt && !machineReadablePaperReceipt && !email && !ftam) {
+        throw new Error(`Expected that institution accepts either paper receipts or sent via email or ftam`)
```

Comment:
FYI @michaelkamphausen :  Most accept email, but there is a handful of institutions that only have a FTAM address.